pyproject.toml

```diff
@@ -3,7 +3,7 @@ line-length = 100
 
 [tool.poetry]
 name = "dbt-copilot-tools"
-version = "0.1.136"
+version = "0.1.138"
```

Comment:
Why the double bump?

Reply:
I think it was when I merged in main. Probably manually bumped the version after the hook had already bumped.